dataminer-overview/Support Services/Support_services_agreement.md

```diff
@@ -69,13 +69,13 @@ The User shall ensure that they have an industry-standard and uninterrupted remo
 
 ## Ticket generation
 
-When the User encounters an Issue with the Software or wishes to receive Support for another reason, the User can at their choice [open a ticket for Support or call Skyline's Support helpdesk](https://skyline.be/contact/tech-support).
+When the User encounters an Issue with the Software or wishes to receive Support for another reason, the User can at their choice [open a ticket for Support or call Skyline's Support helpdesk](https://operations.skyline.be/).
 
-Tickets for Support can be generated through dataminer.services.
+Tickets for Support can be generated through the [User Operations Support](xref:User_operations_support) web application.
 
 Support is available during Business Hours only, except if the User has purchased an Out of Business Hours Support plan. If the User wishes to create a ticket in follow-up of their telephone support, the User needs to generate a ticket through Skyline's dataminer.services.
 
-The User can always consult the status of their tickets on Skyline's dataminer.services. The ticket number will be used by Skyline for the entire life cycle of the specific Issue or question.
+The User can always consult the status of their tickets using Skyline's [User Operations Support](xref:User_operations_support) web application. The ticket number will be used by Skyline for the entire life cycle of the specific Issue or question.
```

Comment:
+The User can always consult the status of their tickets using Skyline's [User Operations Support web application](https://operations.skyline.be/), more details can be found on [User Operations Support documentation](xref:User_operations_support). 
The ticket number will be used by Skyline for the entire life cycle of the specific Issue or question.